gossip/src/crds_value.rs

```diff
@@ -91,6 +91,7 @@ pub enum CrdsData {
     Version(Version),
     NodeInstance(NodeInstance),
     DuplicateShred(DuplicateShredIndex, DuplicateShred),
+    IncrementalSnapshotHashes(IncrementalSnapshotHash),
```

Comment:
Why one is `IncrementalSnapshotHashes`, the other is `IncrementalSnapshotHash`?

If you are following `SnapshotHashes`, I don't think that is a good example to follow.

Reply:
Done. I've renamed the struct to `IncrementalSnapshotHashes`.